File: withFunctions/findSmallestAndBiggest.cpp
#include<iostream>
using namespace std;
int smallest(int x, int y){
	if(x > y){
		return y;
	}
	else{
		return x;
	}
}
int biggest(int x, int y){
	if(x > y){
		return x;
	}
	else{
		return y;
	}
}
int main(){
	int x, y;
	cout << "Enter two numbers = ";
	cin >> x >> y;
	int smallN = smallest(x, y);
	int bigN = biggest(x, y);
	cout << "Small number = " << smallN << endl << "Big number = " << bigN;
	return 0;
}

File: withFunctions/findSmallestAndBiggest.java
package springIntro;

import java.util.*;

public class findSmallestAndBiggest {

	static int biggest(int x, int y) {
		if(x > y)
			return x;
		else 
			return y;
	}
	static int smallest(int x, int y) {
		if(x > y)
			return y;
		else 
			return x;
	}
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		System.out.println("Enter two number");
		int x = in.nextInt();
		int y = in.nextInt();
		System.out.println("Small number = " + smallest(x, y));
		System.out.println("Big number = " + biggest(x, y));
		
	}

}
